contributing.md

```diff
@@ -1,3 +1,10 @@
+### Follow Conventional Commits
+
+This repository enforces
+[Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/). For
+writing pull request titles and as well as issues. Please follow the convention
+when writing them.
```

Comment:
Please change the wording to make the sentence more grammatical correct.

> This repository enforces
[Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/). For
writing pull request titles and as well as issues.

to

> This repository enforces the
[Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/) specification for
for composing titles for pull requests as well as for issues.